no-computer.md

```diff
@@ -46,6 +59,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are many DSLs in this domain, such as soccer, Monopoly, volleyball, and Super Smash Brothers. Melee.  These all address the needs of users relatively well (except for Monopoly) by providing a skill-based manner in which people's competitive tendencies can be appeased.  An admirable quality that many of these DSLs have is that they have well-defined rules, which is something that pingminton could learn from.  Most could be improved by making them more accessible (i.e. involving less equipment).
```

Comment:
I wouldn't quite put monopoly as a way to exert dominance... more as a way to slowly lose friends when they land on Boardwalk.